catalog/pages/banner/index.md

```diff
@@ -0,0 +1,134 @@
+# Feedback banner
+
+## Props
+
+```table
+span: 6
+rows:
+  - Prop: isOpen
+    Type: boolean
+    Default: "false"
+    Notes: Defines if banner is open or not. Required prop.
+  - Prop: isExpanded
+    Type: boolean
+    Default: "false"
+    Notes: Allows to open banner already expanded.
+  - Prop: onRequestClose
+    Type: function
+    Default: none
+    Notes: A callback function which gets executed when the close button is pressed.  No close button would be visible if this prop is not passed.
+  - Prop: heading
+    Type: string
+    Default: none
+    Notes: Heading text. Required prop.
+  - Prop: content
+    Type: string
+    Default: none
+    Notes: Content text
+  - Prop: linkText
+    Type: string
+    Default: none
+    Notes: Link text. Should be used with href.
+  - Prop: href
+    Type: string
+    Default: none
+    Notes: Specifies the URL of the page the link goes to. When `href` passed control element is an anchor element otherwise it is the expand/collapse button. Should NOT be used with onButtonClick, expandedText or collapsedText. Should be used with linkText
+  - Prop: linkProps
+    Type: string
+    Default: none
+    Notes: Link props when link control element is used. e.g. `target`
+  - Prop: onButtonClick
+    Type: function
+    Default: none
+    Notes: A callback function which gets executed when the expand/collapse button is clicked. This prop should NOT be used with href or linkText and should be passed when expand/collapse button is needed.
+  - Prop: expandedText
+    Type: string
+    Default: none
+    Notes: Expanded state control button text. Should be used with collapsedText and onButtonClick.
+  - Prop: collapsedText
+    Type: string
+    Default: none
+    Notes: Collapsed state control button text. Should be used with expandedText and onButtonClick.
+  - Prop: buttonProps
+    Type: string
```

Comment:
I think `linkProps` and `buttonProps` should be type `object`

Reply:
Sure. Fixed.